CONTRIBUTING.md

```diff
@@ -28,6 +28,15 @@ Build the `temporal` binary:
 go build ./cmd/temporal
 ```
 
+## Generate docs
+
+```bash
+go build -o temporal-doc-gen ./cmd/docgen
```

Comment:
I think there's a target in the Makefile (which I'm totally fine with removing BTW).

Another option is running with `go run`.

```
go run ./cmd/docgen
```

Reply:
I'd switch to `go run` though.